src/components/DayPickerRangeController.jsx

```diff
@@ -1013,7 +1025,7 @@ export default class DayPickerRangeController extends React.PureComponent {
           ...days,
           [monthIso]: {
             ...month,
-            [iso]: modifiers,
+            [iso]: this.getCachedModifiers(Array.from(modifiers)),
```

Comment:
why is Array.from needed here?

Reply:
modifiers is a Set here, and this.getCachedModifiers expects an array, not a Set, because it calls .join on it to generate the cache key.